mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -338,8 +338,10 @@ function render_global_header( $attributes = array() ) {
 
 	// Preload the menu font.
 	if ( is_callable( 'global_fonts_preload' ) ) {
-		if ( ! is_rosetta_site() ) {
-			global_fonts_preload( 'Inter', 'latin' );
+		/* translators: Subsets can be any of cyrillic, cyrillic-ext, greek, greek-ext, vietnamese, latin, latin-ext.  */
+		$subsets = explode( ',', _x( 'Latin', 'Global menu font subsets, comma separated', 'wporg' ) );
```

Comment:
We should keep this lowercase just for consistency.

```suggestion
		$subsets = explode( ',', _x( 'latin', 'Global menu font subsets, comma separated', 'wporg' ) );
```